nengo_bones/templates/.pre-commit-config.yaml.template

```diff
@@ -9,7 +9,7 @@ repos:
       exclude: |
           (?x)(
               {% for pattern in exclude %}
-              {% if not loop.first %}| {% endif %}{{ pattern }}
+              {{ "" }}{% if not loop.first %}| {% endif %}{{ pattern }}
```

Comment:
What is the `{{ "" }}` for?

Reply:
For some reason the `if` block doesn't preserve indentation, so the `{{ "" }}` is just kind of a noop (which will be indented), and then the if block is anchored against that.

Reply:
:+1: